lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -289,12 +292,19 @@ def add_to_list_top
             self[position_column] = acts_as_list_top
           end
 
+          # A poorly named method. It will insert the item at the desired position if the position
+          # has been set manually using position=, not necessarily the bottom of the list
           def add_to_list_bottom
-            if not_in_list? || scope_changed? && !@position_changed || default_position?
+            if not_in_list? || internal_scope_changed? && !@position_changed || default_position?
               self[position_column] = bottom_position_in_list.to_i + 1
+              @scope_changed = false
             else
               increment_positions_on_lower_items(self[position_column], id)
+              @scope_changed = false
```

Comment:
Since we're setting this in both circumstances, you can move `@scope_changed = false` to below the conditional.